Description: fix FTBFS with automake 1.12 or later.
 Declared tests as serial because default behaviour is 
 parallel test harness.
Bug-Debian: http://bugs.debian.org/712548
Author: Eugene Seliverstov <theirix@gmail.com>
Last-Update: 2013-06-17
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/configure.ac
+++ b/configure.ac
@@ -1,5 +1,5 @@
 AC_INIT([asn1c], [0.9.24], [vlm@lionet.info])
-AM_INIT_AUTOMAKE([-Wall -Werror foreign])
+AM_INIT_AUTOMAKE([-Wall -Werror foreign serial-tests])
 AC_CONFIG_HEADER([config.h])
 AC_CONFIG_MACRO_DIR([m4])

